steampy/client.py

```diff
@@ -29,21 +31,47 @@ def func_wrapper(self, *args, **kwargs):
 
 
 class SteamClient:
-    def __init__(self, api_key: str, username: str = None, password: str = None, steam_guard:str = None) -> None:
+    def __init__(self, api_key: str, username: str = None, password: str = None, steam_guard: str = None, cookies: dict = None, proxies: dict = None, autologout: bool = True) -> None:
         self._api_key = api_key
+        self._autologout = autologout
         self._session = requests.Session()
-        self.steam_guard = steam_guard
+        if proxies:
+            if not isinstance(proxies, dict) :
+                raise TypeError('proxies must be a dict. Example: \{"http": "http://login:password@host:port"\, "https": "http://login:password@host:port"\}')
+            
+            # ping to steamcommunity.com
+            proxy_status = ping_proxy(proxies)
+
+            if proxy_status is True:
+                self._session.proxies.update(proxies)
+
+        self.steam_guard = guard.load_steam_guard(steam_guard)
```

Comment:
I dont like it readed here, it is loaded in login function and thus SteamClient can be used for some cases with API key and without guard. Guard can be hard to obtain so this is braking change.

Reply:
fixed